internal/repository/prisma/dbsqlc/events.sql

```diff
@@ -3,12 +3,28 @@ SELECT
     count(*) OVER() AS total
 FROM
     "Event" as events
+LEFT JOIN
+  "WorkflowRunTriggeredBy" as runTriggers ON events."id" = runTriggers."eventId"
+LEFT JOIN
+  "WorkflowRun" as runs ON runTriggers."parentId" = runs."id"
+LEFT JOIN
+  "WorkflowVersion" as workflowVersion ON workflowVersion."id" = runs."workflowVersionId"
+LEFT JOIN
+  "Workflow" as workflow ON workflowVersion."workflowId" = workflow."id"
 WHERE
-    events."tenantId" = $1 AND
-    (
-        sqlc.narg('keys')::text[] IS NULL OR
-        events."key" = ANY(sqlc.narg('keys')::text[])
-    );
+  events."tenantId" = $1 AND
+  (
+    sqlc.narg('keys')::text[] IS NULL OR
+    events."key" = ANY(sqlc.narg('keys')::text[])
+    ) AND
+  (
+    sqlc.narg('workflows')::uuid[] IS NULL OR
+    workflow."id" = ANY(sqlc.narg('workflows')::uuid[])
+    ) AND
+  (
+    sqlc.narg('search')::text IS NULL OR
+    jsonb_path_exists(events."data", cast(concat('$.** ? (@.type() == "string" && @ like_regex "', sqlc.narg('search')::text, '")') as jsonpath))
+  );
```

Comment:
I figured we need to return the count of the current search, so this should also include all the existing and the new filter. Is this correct? @abelanger5

Reply:
Yep, that's right 👍